src/components/RecommendedRepoList.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {FloatRight, FloatLeft, Flex, FlexHeader, FlexCenter} from "../styles/Grid";
+import {plus} from "../icons";
+import {truncate} from "../lib/truncate";
+import Avatar from "../styles/Avatar";
+import api from "../lib/apiGraphQL";
+
+function RecommendedRepoList({goal, goalsId}) {
+  const _handleGoalCreation = async(goal) => {
+
+    const repoUrl = `https://github.com/${goal}`
+
+    api
+      .createGoal(goalsId, repoUrl, null)
+      .then(response => {
+        console.log(response)
+      })
+      .catch(e => console.error(e));
+  };
+
+  return (
+    <div>
+      <FlexHeader>
+        <FloatLeft>
+          <FlexCenter>
+            <Avatar
+              small
+              alt="avatar"
+              src={`https://avatars.githubusercontent.com/${goal.nameWithOwner.split("/")[0].replace(/\s+/g, "")}`}
+            />
+            <Flex className="details">
+              <p>{truncate(goal.nameWithOwner, 60)}</p>
+            </Flex>
+          </FlexCenter>
+        </FloatLeft>
+        <FloatRight>
+          <FlexCenter>
+            <a onClick={_handleGoalCreation(goal.nameWithOwner)} href="#"><img alt="pointing right icon" src={plus} /></a>
```

Comment:
Everytime the page is loaded it adds the rpeo. 
```suggestion
            <a onClick={() => _handleGoalCreation(goal.nameWithOwner)} href="#"><img alt="pointing right icon" src={plus} /></a>
```